winc1500 wifi driver fails to build

**Describe the bug**
I was trying to integrate the winc1500 driver into a project that I was working on and noticed that the build was failing when I enabled the driver. I then tried to to build the `samples/net/wifi` project using the `reel_board` as the target (which uses the winc1500) and it fails with the exact same error.


**To Reproduce**
Steps to reproduce the behavior:
1. cd $ZEPHYR_BASE
2. git checkout v3.0.0
3. west update
4. west build -p -b reel_board samples/net/wifi
5. See error


**Expected behavior**
Sample code to compile without error.


**Impact**
Unable to use the winc1500 module with Zephyr.


**Logs and console output**
```
% west build -p -b reel_board samples/net/wifi
-- west build: making build dir /Users/bmiller/zephyrproject/zephyr/build pristine
-- west build: generating a build system
Including boilerplate (Zephyr base): /Users/bmiller/zephyrproject/zephyr/cmake/app/boilerplate.cmake
-- Application: /Users/bmiller/zephyrproject/zephyr/samples/net/wifi
-- Zephyr version: 3.0.0 (/Users/bmiller/zephyrproject/zephyr)
-- Found Python3: /Users/bmiller/zephyrproject/.venv/bin/python3.9 (found suitable exact version "3.9.10") found components: Interpreter 
-- Found west (found suitable version "0.12.0", minimum required is "0.7.1")
-- Board: reel_board
-- Cache files will be written to: /Users/bmiller/Library/Caches/zephyr
-- Found dtc: /opt/homebrew/bin/dtc (found suitable version "1.6.1", minimum required is "1.4.6")
-- Found toolchain: gnuarmemb (/opt/homebrew)
-- Found BOARD.dts: /Users/bmiller/zephyrproject/zephyr/boards/arm/reel_board/reel_board.dts
-- Found devicetree overlay: /Users/bmiller/zephyrproject/zephyr/samples/net/wifi/boards/reel_board.overlay
-- Generated zephyr.dts: /Users/bmiller/zephyrproject/zephyr/build/zephyr/zephyr.dts
-- Generated devicetree_unfixed.h: /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/devicetree_unfixed.h
-- Generated device_extern.h: /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/device_extern.h
-- Including generated dts.cmake file: /Users/bmiller/zephyrproject/zephyr/build/zephyr/dts.cmake

warning: TEST_RANDOM_GENERATOR (defined at subsys/random/Kconfig:8) was assigned the value 'y' but
got the value 'n'. Check these unsatisfied dependencies: (!ENTROPY_HAS_DRIVER) (=n). See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_TEST_RANDOM_GENERATOR.html and/or look
up TEST_RANDOM_GENERATOR in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.

Parsing /Users/bmiller/zephyrproject/zephyr/Kconfig
Loaded configuration '/Users/bmiller/zephyrproject/zephyr/boards/arm/reel_board/reel_board_defconfig'
Merged configuration '/Users/bmiller/zephyrproject/zephyr/samples/net/wifi/prj.conf'
Merged configuration '/Users/bmiller/zephyrproject/zephyr/samples/net/wifi/boards/reel_board.conf'
Configuration saved to '/Users/bmiller/zephyrproject/zephyr/build/zephyr/.config'
Kconfig header saved to '/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h'
-- The C compiler identification is GNU 10.3.1
-- The CXX compiler identification is GNU 10.3.1
-- The ASM compiler identification is GNU
-- Found assembler: /opt/homebrew/bin/arm-none-eabi-gcc
CMake Warning at /Users/bmiller/zephyrproject/zephyr/CMakeLists.txt:775 (message):
  No SOURCES given to Zephyr library: subsys__net__pkt_filter

  Excluding target from build.


-- Configuring done
-- Generating done
-- Build files have been written to: /Users/bmiller/zephyrproject/zephyr/build
-- west build: building application
[1/325] Preparing syscall dependency handling

[3/325] Generating include/generated/version.h
-- Zephyr version: 3.0.0 (/Users/bmiller/zephyrproject/zephyr), build: zephyr-v3.0.0
[10/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmspi.c:41:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[11/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./driver/include/m2m_wifi.h:53,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_wifi.c:42:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[12/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./driver/include/m2m_periph.h:51,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_periph.c:47:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[13/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmasic.c:42:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[14/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.h:45,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmbus.c:43:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[15/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/m2m_ota.c:47:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[16/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/driver/source/nmdrv.c:42:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[17/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./spi_flash/include/spi_flash.h:111,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/spi_flash/source/spi_flash.c:45:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[18/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./socket/include/socket.h:60,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/socket/source/socket.c:47:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[19/325] Building C object zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c.obj 
ccache /opt/homebrew/bin/arm-none-eabi-gcc -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/bmiller/zephyrproject/zephyr/kernel/include -I/Users/bmiller/zephyrproject/zephyr/arch/arm/include -I/Users/bmiller/zephyrproject/zephyr/include -I/Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/bmiller/zephyrproject/zephyr/lib/util/fnmatch/. -I/Users/bmiller/zephyrproject/zephyr/soc/arm/nordic_nrf/common/. -I/Users/bmiller/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/. -I/Users/bmiller/zephyrproject/zephyr/include/drivers -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/bmiller/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/bmiller/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -isystem /Users/bmiller/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include -isystem /Applications/ARM/bin/../lib/gcc/arm-none-eabi/10.3.1/include-fixed -Os -imacros /Users/bmiller/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /Users/bmiller/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/bmiller/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/bmiller/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c.obj -MF zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c.obj -c /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c
In file included from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_debug.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./common/include/nm_common.h:46,
                 from /Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/common/source/nm_common.c:41:
/Users/bmiller/zephyrproject/modules/hal/atmel/asf/common/components/wifi/winc1500/./bsp/include/nm_bsp_internal.h:11:10: fatal error: wifi_winc1500_nm_bsp_internal.h: No such file or directory
   11 | #include <wifi_winc1500_nm_bsp_internal.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /opt/homebrew/bin/cmake --build /Users/bmiller/zephyrproject/zephyr/build
```

**Environment (please complete the following information):**
 - OS: MacOS 12.2.1 and Ubuntu 20.04
 - Toolchain: GNU Arm Embedded Toolchain 10.3-2021.10 on MacOS, Zephyr SDK on Ubuntu
 - Commit: tag v3.0.0

**Additional context**
The error message above is from building on MacOS, but I also tried building on Linux and got the same error.

--- a/drivers/wifi/winc1500/CMakeLists.txt
+++ b/drivers/wifi/winc1500/CMakeLists.txt
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if(CONFIG_WIFI_WINC1500)
-  zephyr_library_include_directories(./)
+  zephyr_include_directories(./)
 
   zephyr_library_sources(
     wifi_winc1500_nm_bsp.c

